x/concentrated-liquidity/types/keys.go

```diff
@@ -205,10 +207,10 @@ func KeyPool(poolId uint64) []byte {
 }
 
 // Incentive Prefix Keys
-// KeyIncentiveRecord is the key used to store incentive record struct for the
-// pool id + min uptime index + denom + addr combination.
-func KeyIncentiveRecord(poolId uint64, minUptimeIndex int, denom string, addr sdk.AccAddress) []byte {
-	return []byte(fmt.Sprintf("%s%s%d%s%d%s%s%s%s", IncentivePrefix, KeySeparator, poolId, KeySeparator, minUptimeIndex, KeySeparator, denom, KeySeparator, addr))
+// KeyIncentiveRecord is the key used to store incentive records using the combination of
+// pool id + min uptime index + incentive record id.
+func KeyIncentiveRecord(poolId uint64, minUptimeIndex int, id uint64) []byte {
+	return []byte(fmt.Sprintf("%s%s%d%s%d%s%d", IncentivePrefix, KeySeparator, poolId, KeySeparator, minUptimeIndex, KeySeparator, id))
```

Comment:
To prevent this from ever turning into a key malleability bug due to the hanging int, could we suffix with a key separator?